Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -2379,6 +2375,74 @@ bool CLuaEngineDefs::EngineGetModelFlag(uint uiModelId, eModelIdeFlag eFlag)
     return pModelInfo->GetIdeFlag(eFlag);
 }
 
+bool CLuaEngineDefs::EngineSetModelFlags(uint uiModelID, uint uiFlags, std::optional<bool> bIdeFlags)
```

Comment:
`std::uint32_t` instead of `uint`

Reply:
i didn't touch anything in here, looks like the code messed up with master, I gonna need to clean it a bit up